src/ResourceManager/Sql/Commands.Sql/Database/Cmdlet/CancelAzureSqlDatabaseActivity.cs

```diff
@@ -0,0 +1,54 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Sql.Database.Model;
+using System.Collections.Generic;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Sql.Database.Cmdlet
+{
+    [Cmdlet(VerbsLifecycle.Stop, "AzureRmSqlDatabaseActivity", SupportsShouldProcess = true,
+        ConfirmImpact = ConfirmImpact.None)]
+    public class CancelAzureSqlDatabaseActivity : AzureSqlDatabaseActivityCmdletBase
```

Comment:
@jiawang003 nit: consider renaming this class to `StopAzureSqlDatabaseActivity` to be consistent with the cmdlet name.

Reply:
renamed